client/me/help/live-chat-closure-notice/index.jsx

```diff
@@ -31,13 +31,13 @@ const LiveChatClosureNotice = ( { closesAt, compact, displayAt, holidayName, reo
 		return null;
 	}
 
-	const heading = translate( 'Live chat closed for %(holidayName)s', {
-		args: { holidayName },
-	} );
-
-	let message;
+	let heading, message;
 
 	if ( currentDate.isBefore( closesAt ) ) {
+		heading = translate( 'Live chat will be closed for %(holidayName)s', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Live chat closed for %(holidayName)s' )` *ES Score: 7*